src/backend/query_builder.rs

```diff
@@ -280,7 +280,9 @@ pub trait QueryBuilder: QuotedBuilder + EscapeBuilder {
             }
             SimpleExpr::FunctionCall(func, exprs) => {
                 self.prepare_function(func, sql, collector);
-                self.prepare_tuple(exprs, sql, collector);
+                if !exprs.is_empty() {
```

Comment:
Basically I don't think it is a good practice. If it's a function, we should make sure it's a `FUNCTION()`.